packages/react-reconciler/src/ReactFiber.old.js

```diff
@@ -488,6 +490,10 @@ export function createFiberFromTypeAndProps(
           expirationTime,
           key,
         );
+      case REACT_DEBUG_TRACING_MODE_TYPE:
```

Comment:
Does it mean that if I somehow end up with undefined `type` in a stable release, after this change it would start matching `REACT_DEBUG_TRACING_MODE_TYPE` (which would also be `undefined` in stable), and thus I won't get to the catch-all default case that throws? This seems bad.

Reply:
`REACT_DEBUG_TRACING_MODE_TYPE` isn't undefined in stable, just `React.DebugTracingMode`.

Reply:
(I backed out the conditional symbol export)